example/example.json

```diff
@@ -67,6 +67,13 @@
             "type": "string",
             "title": "Birthdate",
             "format": "date"
+        },
+        "comment": {
+            "type": "string",
+            "title": "comment",
+            "ui:widget": "textarea",
+            "cols": 20,
+            "rows": 3       
```

Comment:
these 2 properties are not part of the standard JSON definition, and also they are related to the UI. Therefore, I suggest to name them `ui:cols` and `ui:rows` (similar to how radio buttons and checkboxes are defined)